### PR TITLE
docs: explain import from parent dirs with example

### PR DESCRIPTION
**Pull request by dmitrika**
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [x] Docs have been added / updated if required

**Description of Changes Made (if issue reference is not provided)**

Faced it myself and it wasn't clear until if it's expected or not. Want to make it clear to users and give an example.

### CODE REVIEW

File: docs/content/Schema/Advanced/Code-Reusability-Export-and-Import.md [hunk 1]
```diff
@@ -88,6 +88,31 @@ cube(`Events`, {
 });
 ```
 
+## Import from parent directories
+
+Cube.js flattens nested directories. Import from parent directories as they are located on the same level.
```
Comment (rpaik) [inline docs/content/Schema/Advanced/Code-Reusability-Export-and-Import.md:93]:
```suggestion
Cube flattens nested directories. Import from parent directories as they are located on the same level.
```

Comment (rpaik) [reply docs/content/Schema/Advanced/Code-Reusability-Export-and-Import.md:93]: Also, are you trying to say you may need to import from parent directories as Cube flattens nested directories? 🙇

Comment (dmitrika) [reply docs/content/Schema/Advanced/Code-Reusability-Export-and-Import.md:93]: @rpaik Yes, indeed, that might be a better wording for what I'm trying to say. Should I swap the whole paragraph?

Comment (rpaik) [reply docs/content/Schema/Advanced/Code-Reusability-Export-and-Import.md:93]: Yes, that works for me @dmitrika 😄

Comment (dmitrika) [reply docs/content/Schema/Advanced/Code-Reusability-Export-and-Import.md]: Pushed updated paragraph version, please take another look.

**Review by rpaik — APPROVED** [review]

**Pull request merged**